CHANGELOG.md

```diff
@@ -1,9 +1,82 @@
+<!--markdownlint-disable MD024 no-duplicate-header-->
 
 # ProGetAutomation Changelog
 
-# 1.1.0
+## 2.0.0
 
-## Added
+### Upgrade Instructions
+
+Updated `Get-ProGetFeed`, `New-ProGetFeed`, `Test-ProGetFeed`, and `Remove-ProGetFeed` functions to use ProGet's
+[Feed Management API](https://docs.inedo.com/docs/proget-reference-api-feed-management) instead of the native API.
+Ensure that API keys/credentials used with those functions have appropriate access to manage feeds.
+
+Update usages of objects returned by `Get-ProGetFeed` and `New-ProGetFeed`. Rename these properties:
+
+* `Active_Indicator` -> `active`

```

Comment:
I like to use ➔ ([Unicode Character U+2794](https://www.compart.com/en/unicode/U+2794))

Reply:
Do you have an extension or keybinding for that? Usually I use Win+., but think it's annoying.

Reply:
Yes I use AutoHotKey [Hotstrings](https://www.autohotkey.com/docs/v1/Hotstrings.htm) to do text expansion. For this character I have mapped `{backtick}{backtick}rightb{space}` which results in AutoHotKey replacing that sequence with the ➔ character.

Here's my current list of hotstrings for reference:

```ahk
; ```` == two typed backticks (a backtick is an AHK escape character)
::````check::✓
::````enter::⏎
::````right::→
::````rightb::➔
::````rb::➔
::````left::←
::````up::↑
::````down::↓
::````tm::™
::````em::—
```